### PR TITLE
Add openjdk repository information to testenv.properties

### PR DESCRIPTION
**Pull request by psaunderualberta**

### CODE REVIEW

**Comment by llxia** [comment]
Thanks @psaunderualberta . Could you squash all 50 commits into one? https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
Please move the PR out of the draft when you are ready. Thanks.

**Comment by psaunderualberta** [comment]
[Grinder](https://ci.adoptopenjdk.net/job/Grinder/2759/)

File: openjdk/build.xml [hunk 1]
```diff
@@ -324,14 +324,16 @@
 			</else>
 		</if>
 		<checkGitRepoSha repoDir="openjdk-jdk" />
+		<addTestenvProperties repoDir="openjdk-jdk" repoName="JDK${env.JDK_VERSION}_${env.JDK_IMPL^^}"/>
```
Comment (llxia) [inline openjdk/build.xml:327]: Please remove extra `^^`

Comment (psaunderualberta) [reply openjdk/build.xml:327]: The "^^" is used to force the JDK_IMPL variable to be uppercase

**Comment by psaunderualberta** [comment]
[Grinder](https://ci.adoptopenjdk.net/job/Grinder/2820)

**Comment by llxia** [comment]
Close this PR as the change is delivered via https://github.com/adoptium/aqa-tests/pull/3196 and https://github.com/adoptium/TKG/pull/256.

**Pull request closed without merging**